common/src/View/CompilationRunner.cpp

```diff
@@ -172,6 +172,50 @@ void CompilationCopyFilesTaskRunner::doExecute()
 
 void CompilationCopyFilesTaskRunner::doTerminate() {}
 
+CompilationDeleteFilesTaskRunner::CompilationDeleteFilesTaskRunner(
+  CompilationContext& context, const Model::CompilationDeleteFiles& task)
+  : CompilationTaskRunner{context}
+  , m_task{task.clone()}
+{
+}
+
+CompilationDeleteFilesTaskRunner::~CompilationDeleteFilesTaskRunner() = default;
+
+void CompilationDeleteFilesTaskRunner::doExecute()
+{
+  emit start();
+
+  try
+  {
+    const auto targetPath = IO::Path{interpolate(m_task->targetSpec())};
+
+    const auto targetDirPath = targetPath.deleteLastComponent();
+    const auto targetPattern = targetPath.lastComponent().asString();
+
+    try
+    {
+      m_context << "#### Deleting '" << IO::pathAsQString(targetPath) << "'\n";
```

Comment:
I think it would be better to explicitly log the files that will get deleted after the pattern was resolved.

Reply:
Want me to make that change for the CompilationCopyFilesTaskRunner logging while I'm at it?

Reply:
Yep, that would be great. Thanks!

Reply:
Done.